docker/compose/docker-compose-nfs.yml

```diff
@@ -0,0 +1,91 @@
+#
```

Comment:
`docker-compose-nfs.yml` and `docker-compose-nfs-mount.yml` follow the same pattern as `docker-compose.yml`. The use case is same. Which I think is to allow users to deploy a small cluster easily. This PR is not changing the way bookies are deployed in the original `docker-compose.yml`.